index.html

```diff
@@ -3,7 +3,7 @@
         <title>My website</title>
```

Comment:
Sorry, apparently that code academy link is old. Here's the newer course: https://www.codecademy.com/learn/learn-html-css